src/main/webapp/WEB-INF/views/cars/all.jsp

```diff
@@ -0,0 +1,39 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>All cars</title>
+</head>
+<body>
+<%@include file="../header.jsp" %>
+<h1>All cars</h1>
+    <table border="1">
+        <tr>
+            <td>Id</td>
+            <td>Model</td>
+            <td>Manufacturer name</td>
+            <td>Manufacturer country</td>
+            <td>Drivers</td>
+            <td colspan="2">Setting</td>
+        </tr>
+        <c:forEach items="${cars}" var = "car">
+            <tr>
+                <td><c:out value="${car.id}"/></td>
+                <td><c:out value="${car.model}"/></td>
+                <td><c:out value="${car.manufacturer.name}"/></td>
+                <td><c:out value="${car.manufacturer.country}"/></td>
+                <td>
+                    <c:if test="${car.drivers.size() == 0}">free</c:if>

```

Comment:
nice feature, but "free" sounds like "free to buy/get". better write here "no drivers" or something.